quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -213,6 +213,17 @@ impl Limits {
     );
     setter!(with_ack_elicitation_interval, ack_elicitation_interval, u8);
     setter!(with_max_ack_ranges, ack_ranges_limit, u8);
+    /// Sets the maximum send buffer size for a Stream
```

Comment:
Unfortunately you can't comment on items emitted by macros. We'll have to move the comment _into_ the macro invocation and match on `$(#[doc = $doc:literal])*`

Reply:
```rust

macro_rules! setter {
    ($(#[doc = $doc:literal])* $name:ident, $field:ident, $inner:ty $(, |$validate_value:ident| $validaiton:block)?) => {
        $(#[doc = $doc])*
        pub fn $name(mut self, value: $inner) -> Result<Self, ValidationError> {
            $(
                let $validate_value = value;
                $validaiton
            )?
            self.$field = value.try_into()?;
            Ok(self)
        }
    };
}
```

Reply:
gave me a chance to fix a typo that was bugging me :-)

Reply:
I will also note I was able to figure out the crazy macro syntax on my own before I saw your suggestion :-D

Reply:
https://dnglbrstg7yg.cloudfront.net/10f733ff877c99d8c04026bd388a84383f2e227a/doc/s2n_quic/provider/limits/struct.Limits.html#method.with_max_send_buffer_size

Reply:
> gave me a chance to fix a typo that was bugging me :-)

I'm surprised the `typos` check didn't catch that...